opencog/atoms/execution/Instantiator.cc

```diff
@@ -358,8 +358,10 @@ Handle Instantiator::walk_tree(const Handle& expr, bool silent)
 			{
 				args = beta_reduce(args, *_vmap);
 			}
-
-			const HandleSeq& oset(args->getOutgoingSet());
+			const Type& arg_type = args->get_type();
+			// unpack list link
+			const HandleSeq& oset(arg_type == LIST_LINK ? args->getOutgoingSet():
+															HandleSeq{args});
```

Comment:
That line is strangely aligned, that is why smart tabs (also called mixed tabs) exists! 

See https://www.emacswiki.org/emacs/SmartTabs or https://github.com/opencog/atomspace/pull/1643#discussion_r184526590 for more info.

Smart/Mixed tabs is not mandatory but it is recommended and is likely supported by your editor.

Reply:
Thanks!